rpcs3/Emu/System.cpp

```diff
@@ -918,7 +918,14 @@ game_boot_result Emulator::Load(const std::string& title_id, bool is_disc_patch,
 				// Load /dev_bdvd/ from game list if available
 				if (std::string game_path = m_games_config.get_path(m_title_id); !game_path.empty())
 				{
-					disc = std::move(game_path);
+					if (fs::is_file(game_path + "/PS3_DISC.SFB"))
+					{
+						disc = std::move(game_path);
+					}
+					else
+					{
+						inherited_ps3_game = std::move(game_path);
```

Comment:
I'm missing the .clear() call somewhere

Reply:
Actually, it seems like there is no reason for it to be a member anymore I think

Reply:
Yeah I made it a local.